subsys/usb/host/usbh_core.c

```diff
@@ -71,6 +75,11 @@ static void dev_connected_handler(struct usbh_context *const ctx,
 	if (usbh_device_init(ctx->root)) {
 		LOG_ERR("Failed to reset new USB device");
 	}
+
+	ret = usbh_class_probe_all(ctx->root);
+	if (ret != 0) {
+		LOG_ERR("Failed to probe all classes for this new USB device");
+	}
 }
 
 static void dev_removed_handler(struct usbh_context *const ctx)
```

Comment:
Will you call `usbh_class_remove_all` in this pr?

Reply:
This must have been an omission, thank you for the reminder.